### PR TITLE
[doc] [example] Example of a setup with multiple workspaces.

### PR DESCRIPTION
**Pull request by ejgallego**
Thanks to Arthur Charguéraud for the example.

### CODE REVIEW

**Comment by ejgallego** [comment]
@charguer any feedback on this ?

**Comment by charguer** [comment]
I've been very busy lately, I should have time second half of next week to try lsp and this multiple workspace stuff.

**Comment by ejgallego** [comment]
Thanks for info, and for the testing!

File: examples/multiple_workspaces/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+including the auto-update. You can also do:
+
+```sh
+$ coqc -R bar bar bar/barx.y
+$ coqc -R foo foo foo/foox.y
+```
+
+## `coq-lsp` workspace documentation
+
+For each workspace added to a project, `coq-lsp` will try to configure
```
Comment (Blaisorblade) [inline examples/multiple_workspaces/README.md:21]:
```suggestion
One can add multiple folders to a workspace — for instance via the "Add Folder to Workspace" command (or [alternatives](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_adding-folders)).
For each folder added to a workspace, `coq-lsp` will try to configure
```

https://code.visualstudio.com/docs/editor/multi-root-workspaces

Comment (Blaisorblade) [reply examples/multiple_workspaces/README.md:21]: That is, we should probably stick to the same terms as VsCode docs, and the above is an attempt.

Comment (ejgallego) [reply examples/multiple_workspaces/README.md:21]: Yes, thanks a lot for the suggestions, indeed we need to improve the wording in the line you suggested.

**Pull request merged**